pkgs/by-name/ai/aider/package.nix

```diff
@@ -0,0 +1,95 @@
+{
+  lib,
+  python311,
+  fetchFromGitHub,
+  git,
+  portaudio,
+}:
+
+python311.pkgs.buildPythonApplication rec {
+  pname = "aider";
+  version = "0.48.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "paul-gauthier";
+    repo = "aider";
+    rev = "v${version}";
+    hash = "sha256-0m5ZHCfxlOOeUvfQznF5hTCJANCBtrO9rWDudQ+RUxM=";
+  };
+
+  build-system = with python311.pkgs; [
+    pythonRelaxDepsHook
+    setuptools
+  ];
+
+  dependencies = with python311.pkgs; [
+    aiohappyeyeballs
+    backoff
+    beautifulsoup4
+    configargparse
+    diff-match-patch
+    diskcache
+    flake8
+    gitpython
+    grep-ast
+    importlib-resources
+    jsonschema
+    llama-index-core
+    llama-index-embeddings-huggingface
```

Comment:
I may be missing something, but where do these come from? I can't find them in the requirements. I've been using the app without them before and it seems to be the source of issues on darwin: `python3.11-tensorflow-2.13.0` is marked broken.

Reply:
Ah, it's an optional one from the "help" set of extras. Could you make those optional and enabled on `!stdenv.isDarwin`? (or maybe on `!tensorflow.meta.broken`? not sure if that works)

Reply:
Indeed, without these extras, the `/help` option in the app is non-functional. They should be pushed as much as possible without breaking environments like Darwin.